src/components/TermsText.jsx

```diff
@@ -1,28 +1,71 @@
 import React from 'react';
-import { CheckArrowIcon } from "../assets/icons/CheckArrowIcon";
 
 export const TermsText = () => (
-  <section className="w-full bg-bgDark2 mb-12 lg:my-10 lg:mb-14">
-    <div className="flex flex-wrap items-center 2xl:w-[1450px] xl:w-[1300px] w-11/12 mx-auto md:pl-4 xl:pr-16 xl:pl-16">
-      <div className="w-full mb-12 lg:mb-0 xl:pl-8">
+  <section className="w-full bg-bgDark2 mb-12 lg:my-10 lg:mb-14 text-white">
+    <div className="flex flex-wrap items-center mx-auto xl:pr-16 xl:pl-16">
+      <div className="w-full mb-12 lg:mb-0">
         <div className="mx-auto lg:mx-auto">
-          <h2 className="text-3xl font-extrabold">Introduction</h2>
-          <p className="text-lg leading-relaxed mb-8">Welcome to our website. By accessing or using our website, you agree to be bound by these terms and conditions.</p>
-          
-          <h2 className="text-3xl font-extrabold">Use of the Website</h2>
-          <p className="text-lg leading-relaxed mb-8">You agree to use the website only for lawful purposes and in a way that does not infringe the rights of others.</p>
-          
-          <h2 className="text-3xl font-extrabold">Intellectual Property</h2>
-          <p className="text-lg leading-relaxed mb-8">All content on this website, including text, graphics, logos, and images, is the property of our company and is protected by copyright laws.</p>
-          
-          <h2 className="text-3xl font-extrabold">Limitation of Liability</h2>
-          <p className="text-lg leading-relaxed mb-8">We are not liable for any damages arising from the use of this website.</p>
-          
-          <h2 className="text-3xl font-extrabold">Changes to Terms</h2>
-          <p className="text-lg leading-relaxed mb-8">We reserve the right to modify these terms at any time. Your continued use of the website constitutes acceptance of the new terms.</p>
-          
-          <h2 className="text-3xl font-extrabold">Contact Us</h2>
-          <p className="text-lg leading-relaxed mb-8">If you have any questions about these terms, please contact us.</p>
+          <h2 className="text-3xl font-extrabold text-primaryText">Terms and Conditions</h2>
+          <p className="text-lg leading-relaxed mb-8">Last updated: {new Date().toISOString().split('T')[0]}</p>
+
+          <h2 className="text-3xl font-extrabold text-primaryText">1. Platform Nature</h2>
+          <p className="text-lg leading-relaxed mb-8">Angor is a decentralized crowdfunding platform that operates on Bitcoin and Nostr protocols. We do not hold custody of funds or act as an intermediary.</p>
```

Comment:
should we not call it a protocol?

Reply:
I am not sure because we do run a domain that allows to pull a compiled package but we don't have a backend so I am not sure we can call it a platform